tests/components/template/test_lock.py

```diff
@@ -1137,3 +1139,58 @@ async def test_emtpy_action_config(hass: HomeAssistant) -> None:
 
     state = hass.states.get("lock.test_template_lock")
     assert state.state == LockState.LOCKED
+
+
+async def test_setup_config_entry(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+) -> None:
+    """Test the config flow."""
+
+    hass.states.async_set(
+        "sensor.test_state",
+        LockState.LOCKED,
+        {},
+    )
+
+    template_config_entry = MockConfigEntry(
+        data={},
+        domain=template.DOMAIN,
+        options={
+            "name": "My template",
+            "state": "{{ states('sensor.test_state') }}",
+            "lock": [],
+            "unlock": [],
```

Comment:
Are lock and unlock lists?

Reply:
To clarify, fan is optimistic so the field accepts an empty list of actions or a list of actions.  The end result is an optimistic fan that runs the actions that may or may not exist.